test/e2e/framework/pod/wait.go

```diff
@@ -561,7 +561,7 @@ func WaitForPodFailedReason(c clientset.Interface, pod *v1.Pod, reason string, t
 		return false, nil
 	})
 	if waitErr != nil {
-		return fmt.Errorf("error waiting for pod SysctlForbidden status: %v", waitErr)
+		return fmt.Errorf("error waiting for pod failure status: %v", waitErr)
```

Comment:
Good catch. Or you could even plug in the expected reason string...

It looks like this is hardcoded in https://github.com/kubernetes/kubernetes/blob/cd6af9e64b65cb198d633f7c161942b5b9fa155b/test/e2e/framework/pods.go#L260-L270 as "SysctlForbidden" but then is only used in sysctl tests.

Reply:
Yup being used in sysctl tests, so I did not touch it.